README.rst

```diff
@@ -0,0 +1,43 @@
+About
+=====
+pyfixm is a library that contains Python wrappers for the FIXM_ XML Schemas,
+plus the `US NAS extension`_ for the FAA. Currently the library is built for
+FIXM v3.0, as this is what the FAA uses to publish data through SWIM_.
+
+Usage
+=====
+
+.. code-block:: python3
+
+    import pyfixm
+    xml = pyfixm.parse("./fixm_file.xml")
+
+Building pyfixm manually
+========================
+To build ``pyfixm`` either use the suppled ``build-pyfixm`` PyCharm run
```

Comment:
Should we mention that this requires Docker to be installed?

Reply:
Added a little note